R/outputs.R

```diff
@@ -370,7 +374,7 @@ add_output_labels <- function(naomi_output, geometry = FALSE) {
 
   indicators <- naomi_output$indicators %>%
     dplyr::select(area_id, sex, age_group, calendar_quarter, indicator,
-                  mean, se, median, mode, lower, upper)
+                  mean, se, median, mode, lower, upper, data_type)
```

Comment:
Remove the `data_type` column from the `indicators` table. Add that column in the function that combines and filters the indicators and input data.